public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -187,6 +187,19 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     }
     const oldDashboardWrapper = new DashboardModelCompatibilityWrapper(this);
 
+    console.log('activating');
+    this.updateIsEmpty();
+
+    console.log(`(Activating) Body key: ${this.state.body.state.key}`);
+    this._subs.add(this.state.body.subscribeToState(this.updateIsEmpty));
```

Comment:
I think the problem is here, because when the panel editor is active, the scene grid layout (this.state.body) will be deactivated, and anything subscribed to its state will be unsubscribed, so `updateIsEmpty` won't be called. When the scene grid layout is reactivated, we don't resubscribe to its state changes.